tests/D2L.CodeStyle.Analyzers.Test/Specs/ConstantAnalyzer.cs

```diff
@@ -0,0 +1,69 @@
+﻿// analyzer: D2L.CodeStyle.Analyzers.Helpers.ConstantAnalyzer
+
+using D2L.CodeStyle.Annotations;
+
+#region Relevant Types
+namespace D2L.CodeStyle.Annotations
+{
+	public static class Contract
+	{
+		public sealed class ConstantAttribute : Attribute { }
+	}
+}
+#endregion
+
+namespace SpecTests
+{
+	using static Contract;
+
+	public sealed class Types

```

Comment:
Since this `ConstantAnalyzer` is general, I would include examples with non-`string`s as well

Reply:
Added integer and boolean.